JsonLogic.Tests/GithubTests.cs

```diff
@@ -138,4 +139,13 @@ public void Issue313_VarEqualsNull()
 
 		Assert.IsTrue(result.IsEquivalentTo(true));
 	}
+
+	[Test]
+	public void Issue318_CanParseStringAsFloatingPointNumberInAnyCulture()
+	{
+		CultureInfo.CurrentCulture = new CultureInfo("de-AT");
+		var number = JsonValue.Create("3.14").Numberify();
+
+		Assert.AreEqual(3.14d, number);
```

Comment:
Could you wrap this in a try/finally and reset the culture to the value it was before the test ran (capture its current value before changing it and restore it in the finally block)?  That way it doesn't affect tests that run afterward.

Reply:
Fixed as proposed.